### PR TITLE
Improve the documentation/README

### PR DESCRIPTION
**Pull request by ddufresne**
https://isidor.numigi.net/web#id=15502&action=298&model=project.task&view_type=form&menu_id=200

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +114,23 @@ To apply a patch directly from a .patch file instead of a git branch, you may do
 **Special cas of Odoo source code**
```
Comment (foutoucour) [inline README.md]:
```suggestion
**Special case of Odoo source code**
```

**Comment by foutoucour** [comment]
Il faut revoir la version de pytest pour python2.7: https://github.com/pytest-dev/pytest/issues/5519

**Comment by ddufresne** [comment]
@foutoucour done

File: .circleci/config.yml [hunk 2]
```diff
@@ -67,24 +67,6 @@ jobs:
       - store_test_results:
           path: ./log
 
-  python2:
```
Comment (foutoucour) [inline .circleci/config.yml]: 👍

**Pull request merged**